python34-devel

I tried to install brainiak on Centos with Python3 installed (not python34-devel) and not the development version. I got this error:

....
    building 'mpi4py.MPI' extension
    /export/swtools/intel/compilers_and_libraries_2016.3.210/linux/mpi/intel64/bin/mpicc -pthread -Wno-unused-result -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -DHAVE_DLFCN_H=1 -DHAVE_DLOPEN=1 -I/usr/include/python3.4m -c src/MPI.c -o build/temp.linux-x86_64-3.4/src/MPI.o
    In file included from src/MPI.c:4:0:
    src/mpi4py.MPI.c:4:20: fatal error: Python.h: No such file or directory
     #include "Python.h"
                        ^
    compilation terminated.
    error: command '/export/swtools/intel/compilers_and_libraries_2016.3.210/linux/mpi/intel64/bin/mpicc' failed with exit status 1

--- a/README.rst
+++ b/README.rst
@@ -33,7 +33,7 @@ Linux
 
 Install the following packages (Ubuntu 14.04 is used in these instructions)::
 
-    apt install build-essential cmake libgomp1 mpich python3-pip
+    apt install build-essential cmake libgomp1 mpich python3-dev python3-pip
 
 Install updated version of the following Python packages::
 

